app/script/calling/SDPMapper.js

```diff
@@ -124,6 +124,6 @@ z.calling.SDPMapper = {
 
     sessionDescription = sdpLines.join('\r\n');
     const sdpInit = {sdp: sessionDescription, type};
-    return Promise.resolve({iceCandidates, sdp: sdpInit});
```

Comment:
Noticeable change, I decide to remove the promise wrapping of the result.
Since it is not needed here (nothing async), it feels more natural and the api is easier to use